www/include/configuration/configGenerate/xml/postcommand.php

```diff
@@ -38,10 +38,12 @@
 }
 
 require_once realpath(dirname(__FILE__) . "/../../../../../config/centreon.config.php");
-require_once _CENTREON_PATH_.'/www/class/centreonDB.class.php';
-require_once _CENTREON_PATH_.'/www/class/centreonXML.class.php';
-require_once _CENTREON_PATH_.'/www/class/centreonInstance.class.php';
-require_once _CENTREON_PATH_.'/www/class/centreonSession.class.php';
+require_once _CENTREON_PATH_ . '/www/class/centreonDB.class.php';
+require_once _CENTREON_PATH_ . '/www/class/centreonXML.class.php';
+require_once _CENTREON_PATH_ . '/www/class/centreonInstance.class.php';
+require_once _CENTREON_PATH_ . '/www/class/centreonSession.class.php';
+
+$centcorePipe = defined('_CENTREON_VARLIB_') ? _CENTREON_VARLIB_ . "/centcore.cmd" : "/var/lib/centreon/centcore.cmd";
```

Comment:
_CENTREON_VARLIB_ is already defined by the required centreon.config.php line 40
I don't know how it can be not set without throwing a fatal error when centreon.config.php is used ?
IMHO, to check if defined is useless